proto/viam/app/v1/billing.proto

```diff
@@ -14,6 +14,7 @@ service BillingService {
   rpc GetBillingSummary(GetBillingSummaryRequest) returns (GetBillingSummaryResponse);
 }
 
+// Will be deprecated/removed in favor of GetCurrentMonthUsageSummaryResponse
```

Comment:
For the tickets also mark deprecated, could you please create and link a ticket,